packages/devtools_app/lib/src/debugger/controls.dart

```diff
@@ -124,7 +124,7 @@ class _DebuggingControlsState extends State<DebuggingControls>
           child: Container(
             color: visible ? Theme.of(context).highlightColor : null,
             child: DebuggerButton(
-              title: 'Libraries',
+              title: 'Program Explorer',
```

Comment:
I'm not completely sold on this name. Maybe "Source Explorer"? Input here @jacob314 @devoncarew?

Reply:
Can we leave it as Libraries for now? We are just showing more information about the libraries.